### ISSUE TITLE
Banjo-Kazooie Jagged tears across screen

### ISSUE THREAD

**Issue by kev4cards** [gonetz/GLideN64#2147]
When you enter the Furnace Fun game near the end of Banjo-Kazooie, the screen will begin to produce jagged tears (VSync on or off) while standing or moving back and forth along the game tiles. It happens maybe a few times a minute. It seems dependent on a large jumping fireball. It only appears in HLE as far as I can tell. It might be there in LLE as a short one-pixel thick line, but it is really hard to catch. Angrylion plus seems to be free of this problem.

Screenshot:

![GLideN64_Banjo-Kazooie_050](https://user-images.githubusercontent.com/21962903/69895386-54ae7780-12fc-11ea-92f7-6cf9f9633d06.png)

Specs: i7-8750H, GTX 1060, Win 10 1903 Home, Project64 2.4.0-976, 32-bit GlideN64 (appveyor commit 2b5e878)

EDIT: Forgot the savestate: 
[Banjo-Kazooie (U) (V1.1).pj.zip](https://github.com/gonetz/GLideN64/files/3906723/Banjo-Kazooie.U.V1.1.pj.zip)

EDIT2 (Nov. 30th, 2019): The black jagged lines do not appear in the beginning three tiles or the rocky starting platform if the entire game board has been lit/completed (probably anything lit beyond the first honeycomb tile).

**Comment by gonetz** [comment]
As I can see, the black jagged lines problem is common for all HLE plugins. It can be a core issue. Have you tested it with 1964 or mupen64plus?

**Comment by oddMLan** [comment]
when testing with angrylion make sure to disable the VI filters. sometimes they tend to hide gaps, black lines and other artifacts that would be visible otherwise (angrylion-plus allows disabling them)

**Comment by kev4cards** [comment]
I checked with mupen64plus (latest m64p with latest gliden64 added). The problem does not appear. My savestate for that point froze, so I had to work around that. I ended up finding some saves online that worked with mupen64plus (couldn't get my pj64 eep recognized by m64p). So, I suppose I could have some corruption on my Project64 playthrough. Anyways, everything checks out with GlideN64 and the jagged lines are gone.

Unfiltered angrylion-plus was okay, too.
